modules/sdk/gradle-plugins-workspace/src/gradleTest/springContext/build.gradle

```diff
@@ -0,0 +1,41 @@
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+
+allprojects {
+	repositories {
+		maven {
+			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+task runGradleTest
+
+runGradleTest {
+	Project serviceProject = project(":modules:sample:sample-service")
+
+	dependsOn ":modules:sample:sample-service:buildService"
```

Comment:
Lets put the "dependsOn" line as the first thing after `runGradleTest`

Reply:
ok, thanks Greg, will resend the pr